### ISSUE TITLE
Randomised affixes when reforging

### ISSUE THREAD

**Issue by Markusd3vil** [Bryce792/ApotheoticAdditions#11]
Everytime when i reforge it randomises the affixes when it shows for example pinata loot 3.5% but when i click it i get something very random not even something close to what there was.

**Comment by Markusd3vil** [comment]
![image](https://github.com/Bryce792/ApotheoticAdditions/assets/159699240/f1657c6f-187a-45e3-a7f3-a1bf418be730)
![image](https://github.com/Bryce792/ApotheoticAdditions/assets/159699240/2902f865-67bb-4c2e-9018-c994752784f8)
![image](https://github.com/Bryce792/ApotheoticAdditions/assets/159699240/6a158f2c-1a01-45fe-94a5-38b8ba66b277)

This all is on 1.20.1 latest

**Comment by Yetice1** [comment]
Having the same issue

**Comment by thegreywulfe** [comment]
Can confirm having the same issue. Appears to the be something that was [at one point being caused by Apotheosis itself](https://github.com/Shadows-of-Fire/Apotheosis/issues/1145), but was fixed in 7.3.0. At that time, for that version of the issue, it was stated to be caused by a desync that only occurred on dedicated multiplayer servers. I can confirm that I can't replicate the problem in singleplayer.

The only time I was able to "solve" it involved removing Additions completely and also resetting all server config files to let them repopulate automatically. But take that with a grain of salt, our server is an absolute mess and there's no telling if that's meaningful information.

**Comment by Keiron-Beadle** [comment]
Yeah, likewise running into this same problem - does anyone know which server configs I should be removing? I can't see to see a file that would indicate Apotheotic Additions ty

**Comment by Kaidokie** [comment]
I just started getting reports of this issue on my dedicated server. Will try removing the config files, as they are all defaults currently anyway.
